data/plugins/entity/interaction/object.rb

```diff
@@ -0,0 +1,6 @@
+# Catch all object actions and turn the player to
+# the position of the object
+
+on :message, :object_action do |player, message|
+  player.turn_to message.position
```

Comment:
I think we need a method which determines the orientation we need to face and the position we need to face (the center of the object is not always the position it's spawned on). IIRC there is an opcode in the object definitions which holds orientation flags denoting what side of the object is intractable. I believe it's used in the clients route finding algorithm.